.github/workflows/doc.yml

```diff
@@ -3,7 +3,6 @@
 name: "Documentation"
 on:
   push:
-  
```

Comment:
Jonathon recently helped out with an update to the doc action / makefile / etc. for Qserv.  It uses the latest Rubin documentation components and it is nice that it doesn't require an additional container, etc.  You may wish to copy that here?  You can see https://github.com/lsst/qserv/pull/658 if you are interested.